app/utils/data.test.ts

```
@@ -0,0 +1,361 @@
+    expect(checkIsEmpty([])).toBe(true);
+    expect(checkIsEmpty('')).toBe(true);
+    expect(checkIsEmpty(null)).toBe(true);
+    expect(checkIsEmpty(undefined)).toBe(true);
+    // TODO expect(checkIsEmpty(Object.create(null))).toBe(true);
+  });
+
+  it('should return false when object has prototype properties but no own properties', () => {
+    class Proto {
+      constructor() {}
+    }
+    // Proto.prototype.prop = 'value';
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)